docs/_templates/versions.html

```diff
@@ -1,7 +1,7 @@
 {# Add rst-badge after rst-versions for small badge style. #}
 <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="versions">
     <span class="rst-current-version" data-toggle="rst-current-version">
-    <span class="fa fa-book fa-element"> RTD </span>
+    <a href="https://docs.soliditylang.org/_/downloads/en/latest/pdf/">PDF</a>
```

Comment:
How about this?
```suggestion
    {% for type, url in downloads %}
        {% if type == 'PDF' %}
            <a href="{{ url }}">{{ type }}</a>
        {% endif %}
    {% endfor %}
```
Otherwise it will always display the link `latest`, regardless of which version of the docs you visit. I think it might be unexpected. It will also have a link to `docs.soliditylang.org` even if you visit from https://solidity.readthedocs.org (though you could fix that one also by just using `{{ PRODUCTION_DOMAIN }}`).

Reply:
Thanks for the suggestion! Committed the same in the latest push.